typesafety/test_str_format.yml

```diff
@@ -0,0 +1,33 @@
+---
+- case: ok__dot_access
```

Comment:
I wonder if that file shouldn't be called `test_dot_access.yml` or `test_objects_access.yml`. From the looks of it you are trying to verify just that, right?

Reply:
Fine by me, I honestly had no idea how to name that file